packages/conjure-client/src/fetchBridge/__tests__/retryingFetchTests.ts

```diff
@@ -112,6 +112,7 @@ describe("RetryingFetch", () => {
         const originalSetTimeout = window.setTimeout;
         const mockedSetTimeout = jest.fn((callback, timeout) => {
```

Comment:
the function we had before didn't match actual settimeout typings and broke with typescript upgrade